package.json

```diff
@@ -47,6 +47,7 @@
         "electron:start:prod": "npm run webpack:prod && npm run electron-start",
         "electron-start": "node src/electron/start",
         "eject": "react-scripts eject",
+        "release-pr": "./scripts/release-pr.sh",
```

Comment:
'slightly' easier way to setup a release PR: `$ npm run release-pr --from 2.0.0-preview.1 --to 2.0.0-preview.2`